css/style.css

```diff
@@ -0,0 +1,595 @@
+@import url('https://fonts.googleapis.com/css2?family=PT+Serif:wght@400;700&family=Roboto&display=swap');
+html{
+    box-sizing: border-box;
+}
+*, *::before, *::after{
+    box-sizing: inherit;
+}
+body { 
+
+    font-family: 'Roboto', sans-serif;
+    font-style: normal;
+    box-sizing: border-box;
+}
+
+.wrap__site{
+
+    min-width: 375px;
+    max-width: 1920px; 
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Unexpected whitespace at end of line (no-eol-whitespace)](https://app.codacy.com/gh/WebHeroSchool/miniProject/pullRequest?prid=5576209)